File: app/src/main/java/org/mifos/mobile/ui/views/GuarantorDetailView.java
package org.mifos.mobile.ui.views;

/*
 * Created by saksham on 25/July/2018
 */

import org.mifos.mobile.ui.views.base.MVPView;

public interface GuarantorDetailView extends MVPView {
    void guarantorDeletedSuccessfully(String message);
    void showError(String message);
}

File: app/src/main/java/org/mifos/mobile/ui/views/GuarantorDetailView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/*
* Created by saksham on 25/July/2018
*/   interface GuarantorDetailView : MVPView {
    fun guarantorDeletedSuccessfully(message: String?)
    fun showError(message: String?)
}